LondonBibleSociety/BSLet172.xml

```diff
@@ -0,0 +1,95 @@
+            nos. AH/V002910/1 - 448410109</p></editionStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                  licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+                  <msIdentifier>
+                     <repository ref="INS0306BSL"/>
+                     <idno>BFBS Mss 172</idno>
+                     <altIdentifier>
+                        <idno>BSMS 172</idno>
+                     </altIdentifier>
+                     <altIdentifier>
+                        <idno>Cowley XXIV</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+               
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                     <locus from="2ra" to="75vb">2a-75vb</locus>
+                     <title type="incomplete" ref="LIT1309Didesq"/>
+                     <textLang mainLang="gez"/>
+                  </msItem>
+               </msContents>
+                  <history>
+                     <origin>
+                        <origPlace>
+                     </origPlace> 
+                         <origDate></origDate>
+                     </origin>
+                  </history>
+                  <additional>
+                     <adminInfo>
+                        <recordHist>
+                           <source>
+                              <listBibl type="catalogue">
+                                 <bibl>
+                                    <ptr target="bm:Cowley1982BBS"/>
+                                    <citedRange unit="page">74-75</citedRange>
+                                 </bibl>
+                              </listBibl>                  
+                           </source>
+                        </recordHist>
+                        </adminInfo>
+              
+                  </additional>
+            </msDesc>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <langUsage><language ident="en">English</language></langUsage>
```

Comment:
you use "gez" above, so it must be defined here:   `<language ident="gez">Gǝʿǝz</language>`